docs/core/tools/dotnet-sln.md

```diff
@@ -60,17 +81,17 @@ dotnet sln add [-h|--help]
 
 - **`SOLUTION_FILE`**
 
-  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+  The solution file to use. If not specified, the command searches the current directory for one. It fails if there are multiple solution files in the directory.
```

Comment:
I'd suggest combining the last two sentences to make clear that it won't fail if there are multiple solution files but one is specified on the command line.